package.json

```diff
@@ -5,7 +5,7 @@
     "coverage",
     "coveralls"
   ],
-  "version": "3.0.7",
+  "version": "3.0.8",
```

Comment:
Also please never do this @ly-cultureiq. This is for the maintainer's to decide

Reply:
point taken - my apologies